ifrs17/Utils/Queries.ipynb

```diff
@@ -217,14 +217,27 @@
         {
             "cell_type": "code",
             "source": [
-                "public static async Task<Dictionary<string, Dictionary<int, YieldCurve>>> LoadCurrentYieldCurveAsync(this IQuerySource querySource, Args args, ",
-                "\n                                                                                                      IEnumerable<DataNodeData> dataNodes)",
-                "\n{",
-                "\n    var contractualCurrenciesInScope = dataNodes.Select(dn => dn.ContractualCurrency).ToHashSet();",
-                "\n    return (await querySource.LoadCurrentAndPreviousParameterAsync<YieldCurve>(args, ",
+                "public static async Task<Dictionary<string, Dictionary<int, YieldCurve>>> LoadCurrentYieldCurveAsync(this IQuerySource querySource, Args args,",
+                "\n                                                                                                                                IEnumerable<DataNodeData> dataNodes)",
+                "\n{    ",
+                "\n    var currentYieldCurveByGoc = new Dictionary<string,  Dictionary<int, YieldCurve>>();",
+                "\n    ",
+                "\n    foreach (var dn in dataNodes)",
+                "\n    {",
+                "\n        var argsNew = args with {Year = dn.Year, Month = MonthInAYear, Scenario = args.Scenario};",
+                "\n        var loadedYc = await querySource.LoadCurrentAndPreviousParameterAsync<YieldCurve>(argsNew, ",
```

Comment:
args should be used instead of argsNew (remove)

Reply:
removed argsNew